### PR TITLE
[CMIS] Add API to get module power up duration

### PR DESCRIPTION
**Pull request by chiourung**
#### Description

Add API to get module power up and power down duration

### CODE REVIEW

File: sonic_platform_base/sonic_xcvr/api/public/cmis.py [hunk 1]
```diff
@@ -695,6 +695,24 @@ def get_datapath_deinit_duration(self):
...
+    def get_module_up_duration(self):
+        '''
+        This function returns the duration of module power up
+        '''
+        if self.is_flat_memory():
+            return 0
+        duration = self.xcvr_eeprom.read(consts.MODULE_PWRUP_DURATION)
+        return float(duration) if duration is not None else 0
+
+    def get_module_down_duration(self):
```
Comment (prgeor) [inline sonic_platform_base/sonic_xcvr/api/public/cmis.py:707]:
```suggestion
    def get_module_pwr_down_duration(self):
```

File: sonic_platform_base/sonic_xcvr/api/public/cmis.py [hunk 2]
```diff
@@ -695,6 +695,24 @@ def get_datapath_deinit_duration(self):
         duration = self.xcvr_eeprom.read(consts.DP_PATH_DEINIT_DURATION)
         return float(duration) if duration is not None else 0
 
+    def get_module_up_duration(self):
```
Comment (prgeor) [inline sonic_platform_base/sonic_xcvr/api/public/cmis.py:698]:
```suggestion
    def get_module_pwr_up_duration(self):
```

**Review by prgeor — COMMENTED** [review]
@chiourung can you address comments

**Comment by prgeor** [comment]
@chiourung thanks for raising the PR. Can you fix the time duration (currently its hard-coded) of [set_lpmode()](https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/sonic_xcvr/api/public/cmis.py#L950)

**Review by mihirpat1 — APPROVED** [review]

File: sonic_platform_base/sonic_xcvr/api/public/cmis.py [hunk 3]
```diff
@@ -695,6 +695,24 @@ def get_datapath_deinit_duration(self):
...
+
+    def get_module_pwr_down_duration(self):
+        '''
+        This function returns the duration of module power down
+        '''
+        if self.is_flat_memory():
+            return 0
+        duration = self.xcvr_eeprom.read(consts.MODULE_PWRDN_DURATION)
+        return float(duration) if duration is not None else 0
+
```
Comment (prgeor) [inline sonic_platform_base/sonic_xcvr/api/public/cmis.py:715]: @chiourung we are not using these APIs for [set_lpmode](https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/sonic_xcvr/api/public/cmis.py#L957). can you fix that?

Comment (chiourung) [reply sonic_platform_base/sonic_xcvr/api/public/cmis.py:715]: I will raising new PR for set_lpmode

**Review by prgeor — APPROVED** [review]

**Pull request merged**